tests/wpt_urlpattern_tests.cpp

```diff
@@ -375,7 +375,30 @@ TEST(wpt_urlpattern_tests, urlpattern_test_data) {
         // - null
         // - {} // response here.
         auto input_value = parse_inputs_array(inputs);
-        // TODO: Parse "expected_match" field here.
+
+        ondemand::value expected_match_value;
+        if (main_object["expected_match"].get(expected_match_value)) {
+          if (expected_match_value.type() ==
+              simdjson::ondemand::json_type::string) {
+            std::string_view expected_match_str;
+            if (expected_match_value.get_string().get(expected_match_str)) {
```

Comment:
```suggestion
            ASSERT_FALSE(expected_match_value.get_string().get(expected_match_str));
```

Reply:
We can replace this if statement with this. no need to check once again.

Reply:
thanks, I changed